config/environment.default.json

```diff
@@ -0,0 +1,26 @@
+{
+  "production": false,
+  // Location of REST API
```

Comment:
Yes, I write a note in documentation to specify this, otherwise could we save template as JS file?